chainer_chemistry/datasets/molnet/molnet_config.py

```diff
@@ -0,0 +1,282 @@
+import chainer.functions as F
+from chainer_chemistry.datasets.molnet.chembl_tasks import chembl_tasks
+from chainer_chemistry.datasets.molnet.toxcast_tasks import toxcast_tasks
+
+molnet_base = 'http://deepchem.io.s3-website-us-west-1.amazonaws.com/datasets/'
+
+molnet_default_config = {
+    "bace": {
+        "dataset_type": 'one_file_csv',
+        "url": molnet_base + 'bace.csv',
+        "smiles_columns": 'mol',
+        "metrics": None,
+        "loss": None,
+        "task_type": 'mix',
+        # pIC50: regression
+        # Class: classification
+        "tasks": ["pIC50", "Class"],
```

Comment:
maybe we can separate "bace_pIC50" dataset type and "bace_Class" dataset type.
As long as "url" is same, I guess we can share the download cache :).

Reply:
👍

Reply:
after the discussion, we conluded that "mix" type should be treated as "mix" - multi-task formulation.